### PR TITLE
bar graph vignette to demonstrate new list/search methods

### PR DESCRIPTION
**Pull request by mountainMath**
use of new list_regions, list_vecters and search_vectors methods

### CODE REVIEW

**Comment by dshkol** [comment]
I think there's a few small changes that can be made to tidy this up a bit. I'll try to make those adjustments when I can find a little bit of time.

**Comment by atheriel** [comment]
Really glad to see this! I was thinking we needed one. I'll take a look at the content tonight.

**Comment by dshkol** [comment]
I'm having lots of errors trying to run this now. May be the types of errors you were having before @mountainMath where line-by-line execution and console execution works, but knitting does not.

**Comment by mountainMath** [comment]
Hmm. It compiles through just fine. Cleared environment and restarted R and did "Run All" and it went through without a hitch.

You will have to load the API key, manually, and you will have to load can census before doing that manually. You can just do that in the console before running the chunk.

File: vignettes/dwelling_type_bar_graph.Rmd [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+### Dynamically Select Regions
+We select the top 6 CMAs by population.
+```{r, echo=TRUE, message=FALSE, warning=FALSE}
+# grab 6 largest CMAs
+cma_list <- cancensus.list_regions(dataset) %>% 
+  filter(level=='CMA') %>% 
+  top_n(6, pop) %>%
+  pull(region)
+regions=list(CMA=cma_list)
+```
```
Comment (atheriel) [inline vignettes/dwelling_type_bar_graph.Rmd]: Should be able to use the new `as_region_list()` function here.

**Comment by dshkol** [comment]
I've proposed a lot of changes to the code here in #56

File: vignettes/dwelling_type_bar_graph.Rmd (hunk 1 above)
Comment (mountainMath) [inline vignettes/dwelling_type_bar_graph.Rmd]: Yes, skip the `pull(region)` line and replace the `regions=list....` with the `as_region_list(cma_list)`.

Comment (dshkol) [inline vignettes/dwelling_type_bar_graph.Rmd]: I missed this in #57

**Pull request merged**